docs/framework/configure-apps/file-schema/wcf/bindings.md

```diff
@@ -33,10 +43,11 @@ This section holds a collection of standard and custom bindings. Each entry is a
  Custom bindings are identified by their `name` attribute. For more information on custom bindings, see [Custom Bindings](../../../../../docs/framework/wcf/extending/custom-bindings.md).  
   
 ## See also
-- <xref:System.ServiceModel.Configuration.BindingsSection>
-- <xref:System.ServiceModel.Channels.Binding>
-- <xref:System.ServiceModel.Channels.BindingElement>
-- [Bindings](../../../../../docs/framework/wcf/bindings.md)
-- [Custom Bindings](../../../../../docs/framework/wcf/extending/custom-bindings.md)
-- [\<customBinding>](../../../../../docs/framework/configure-apps/file-schema/wcf/custombinding.md)
+
+- <xref:System.ServiceModel.Configuration.BindingsSection?displayProperty=nameWithType>  
+- <xref:System.ServiceModel.Channels.Binding?displayProperty=nameWithType>  
+- <xref:System.ServiceModel.Channels.BindingElement?displayProperty=nameWithType>  
+- [Bindings](../../../../../docs/framework/wcf/bindings.md)  
+- [Custom Bindings](../../../../../docs/framework/wcf/extending/custom-bindings.md)  
+- [\<customBinding>](../../../../../docs/framework/configure-apps/file-schema/wcf/custombinding.md)  
 - [\<binding>](../../../../../docs/framework/misc/binding.md)
```

Comment:
you should see if you have more incoming links like this to the old topic and update them to point to this new one. This one needs to be removed though.